EAD-Assignment1.html

```diff
@@ -0,0 +1,487 @@
+                document.getElementById('YearlyCost').innerHTML = convertToUSD(currencyArray[dataRow][2]);
+            }
+            if (currency == "YEN") {
+                document.getElementById('DailyCost').innerHTML = convertToYEN(currencyArray[dataRow][0]);
+                document.getElementById('MonthlyCost').innerHTML = convertToYEN(currencyArray[dataRow][1]);
+                document.getElementById('YearlyCost').innerHTML = convertToYEN(currencyArray[dataRow][2]);
+            }
+            showTotal();
+        }
+        //This function show totol according to selected plan
+        function showTotal() {
+            if (document.getElementById('DailyPlan').checked == true) {
+                document.getElementById('TotalCost').value = document.getElementById('DailyCost').innerHTML;
+            }
+            else if (document.getElementById('MonthlyPlan').checked == true) {
+                document.getElementById('TotalCost').value = document.getElementById('MonthlyCost').innerHTML;
+            }
+            else if (document.getElementById('YearlyPlan').checked == true) {
+                document.getElementById('TotalCost').value = document.getElementById('YearlyCost').innerHTML;
+            }
+        }
+        //This function submit confirm details form
+        function submitConfirmDetails() {
+            document.getElementById('confirmDetailsForm').submit();
+            alert('Registration Successfull');
+        }
+        //This function cancel the registration
+        function cancelRegistration() {
+            document.getElementById('confirmDetailsForm').submit();
+            alert('Registration Cancelled');
+        }
+        //This function convert INR to USD
+        function convertToUSD(value) {
+            return (Math.round((value / 72) * 100) / 100);
+        }
+        //This function convert INR to YEN
+        function convertToYEN(value) {
+            return (Math.round((value * 1.53) * 100) / 100);
+        }
+        //This function hide employee form
+        function hideEmpForm() {
+            document.getElementById('EmployeeRegisterDiv').style.display = "none";
+        }
+        //This function show the vehicle form
+        function showVehicleForm() {
+            document.getElementById('VehicleRegisterDiv').style.display = "inherit";
+        }
+        //This function hide the vehicle form
+        function hideVehicleForm() {
+            document.getElementById('VehicleRegisterDiv').style.display = "none";
+        }
+        //This fuction show the select plan form
+        function showSelectPlanForm() {
+            document.getElementById('SelectPlanDiv').style.display = "inherit";
+        }
+        //This function hide the select plan form
+        function hideSelectPlanForm() {
+            document.getElementById('SelectPlanDiv').style.display = "none";
+        }

```

Comment:
you can write single function as toggleSelectPlanForm and can pass a boolean parameter according to which you can identify the visibility.